src/server/services/api-proxy.js

```diff
@@ -171,26 +178,24 @@ API.createUser = function (email, password) {
   });
 };
 
-API.uploadPicture = function (file, mediaObj) {
-  logger.info('Trying to upload file with id', mediaObj.id);
-  return new Promise(function (resolve, reject) {
-    request
-      .post({
-        url: `${config.media_url}`,
-        headers: { 'X-UPLOAD-TOKEN': mediaObj.uploadToken },
-        formData: {
-          id: mediaObj.id,
-          file: {
-            value: file.buffer,
-            options: {
-              filename: file.originalname,
-              encoding: file.encoding,
-              'Content-Type': file.mimetype,
-              knownLength: file.size
-            }
-          }
-        }
-      }, handleResponse(resolve, reject, 'Successfully uploaded file with id ' + mediaObj.id + ' to backend'));
+API.uploadFile = function (file, fileType = 'image') {
```

Comment:
Did we remove the `DOCUMENT` fileType?

Reply:
No, it is still there. I used image as a default here